hathor/transaction/base_transaction.py

```diff
@@ -172,9 +168,12 @@ def __init__(self,
             Outputs: all outputs that are being created
             Parents: transactions you are confirming (2 transactions and 1 block - in case of a block only)
         """
+        assert version <= 0xFFFF, f'Version {hex(version)} must not be larger than two bytes'
+
         self.nonce = nonce
         self.timestamp = timestamp or int(time.time())
-        self.version = version
+        self.signal_bits = (version >> 8) & 0xFF
```

Comment:
What's the best type for `signal_bits`? We also should explicitly add a `signal_bits: int` at the beginning of the class.

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/623/commits/80814b32d58cd5c0441aec3cb9b10cf124458a78

Reply:
Great! Thanks!